src/polyfill/binding.js

```diff
@@ -0,0 +1,30 @@
+export default {
+  /* jshint expr: true */
+  create (opts) {
+    var elem = document.createElement(opts.extends || opts.id);
+    opts.extends && elem.setAttribute('is', opts.id);
+    return elem;
```

Comment:
Adding the `is` attribute post creation doesn't do anything in native land. This factory never gets run in native, though, it runs the native constructor instead.